lib/src/list.dart

```diff
@@ -16,13 +16,77 @@
 //
 ////////////////////////////////////////////////////////////////////////////////
 
-// ignore_for_file: native_function_body_in_non_sdk_code
-
 import 'dart:core';
 import 'dart:core' as core;
-import 'dart:collection';
+import 'dart:collection' as collection;
+
+import 'native/realm_core.dart';
 
 import 'realm_object.dart';
+import 'realm_class.dart';
+
+class RealmList<T extends Object> extends collection.ListBase<T> {
+  late final RealmListHandle _handle;
+  late final Realm _realm;
+
+  RealmList._(this._handle, this._realm);
+
+  @override
+  int get length => realmCore.getListSize(this);
+
+  //settng lenght is no operation
+  @override
+  set length(int length) {}
```

Comment:
I vaguely remember that set length is used to truncate lists and that some of the methods inherited from ListBase<T> expect this to be true... So I'm not sure it is kosher to make it a no-op. 

But maybe I'm wrong. Just something to check up on

Reply:
will overwrite add instead